tensorflow_recommenders_addons/dynamic_embedding/core/kernels/rocksdb_table_op.h

```diff
@@ -0,0 +1,130 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TFRA_CORE_KERNELS_ROCKSDB_TABLE_H_
+#define TFRA_CORE_KERNELS_ROCKSDB_TABLE_H_
+
+#include "tensorflow/core/kernels/lookup_table_op.h"
+
+namespace tensorflow {
+  namespace recommenders_addons {
```

Comment:
Format code by follow [STYLE_GUIDE](https://github.com/tensorflow/recommenders-addons/blob/master/STYLE_GUIDE.md)

Reply:
@googlebot I signed it!

Reply:
@rhdong Sure will take a look at this.